README.md

```diff
@@ -13,32 +13,36 @@ Repository to describe, develop, document and test the Release Management proces
 * Guidelines and assets for Release Management in CAMARA
 * Describe, develop, document and test the deliverables
 * Started: November 2023
+* First release: r1.0.0 (July 2024)
 
-## Release Information
+## Release Process Information
 
-This projects constains the following (documentation folder):
+This projects constains the following documentation:
 
-* API release guidelines
-* API readiness checklist 
+* [API Release Guidelines](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md)
```

Comment:
I propose to use relative links ... they would still work if we release/tag the repository without a change.

```suggestion
* [API Release Guidelines](/documentation/API_Release_Guidelines.md)
```